src/rules/vertically-aligned-assignments.ts

```diff
@@ -68,13 +72,20 @@ function validateVerticalInitialValueAlignments(stateVariableDeclarationBlocks,
     }
     return errors;
 }
-class VerticallyAlignedAssignments {
-    constructor(reporter) {
+export class VerticallyAlignedAssignments {
```

Comment:
Nit (not blocking to merge this PR)

This can potentially be improved by using [`implements`](https://www.typescriptlang.org/docs/handbook/2/classes.html#class-heritage) here and specifying the abstract class it should follow: this would automatically validate required class parameters (eg `ruleId`, etc) and functional parameters (eg `Reporter`, `ContractDefinition`), so that they don't have to be imported and set manually for every rule. E.g.:

```suggestion
export class VerticallyAlignedAssignments implements SolhintRule {
```

Reply:
I dont think this is trivial if should be fulfill both suggestions. The reason is: the class has to have the function has to be named exactly like the AST element it is executed on. In the case of 
`ContractDefinition` it will only execute on `ContractDefinition` AST elements. So it does not seem possible (i did not find a way) to define a type so that it requires at least one function with the name that matches ASTNode element name.

Reply:
Let's skip this one